htdocs/core/modules/facture/doc/pdf_crabe.modules.php

```diff
@@ -1646,6 +1647,14 @@ protected function _tableau_tot(&$pdf, $object, $deja_regle, $posy, $outputlangs
 		}
 
 		$index++;
+
+		if (getDolGlobalString($conf->global->BILL_TEXT_TOTAL_FOOTER)) {
```

Comment:
Should be
`if (getDolGlobalString('BILL_TEXT_TOTAL_FOOTER')) {`

Reply:
sorry